post-install.yml

```diff
@@ -0,0 +1,146 @@
+- hosts: "{{ hosts_prefix }}-tsuru-api*[0]"
+  sudo: no
+  tasks:
+    - name: Tsuru target-list
+      shell: >
+        tsuru target-list
+      register: tsuru_targets
+    - name: Tsuru add target.
+      shell: >
+        tsuru target-add {{ deploy_env }}_{{ domain_name }} http://127.0.0.1:8080
+      when: "not '{{ deploy_env }}_{{ domain_name }}' in tsuru_targets.stdout"
+    - name: Tsuru set target.
+      shell: >
+        tsuru target-set {{ deploy_env }}_{{ domain_name }}
+    - name: Tsuru login
+      shell: >
+        echo {{ admin_password }} | tsuru login {{ admin_user }}
+
+    - name: Deploy insecure-deployer.pub key
+      copy:
+        src: files/insecure-deployer.pub
```

Comment:
This is the public key, which should be safe to share with the word. Requirement here is that this is the same key as we (or jenkins) have locally, so that we can do git push. Generating new key every time would mean we/jenkins would have to deploy a new key for every new environment...